openy.install

```diff
@@ -70,7 +70,6 @@ function openy_update_8003() {
   // Enable module.
   \Drupal::service('module_installer')->install([
     'config_import',
-    'logger_entity',
```

Comment:
do not remove it!

Reply:
It not used in Open Y code anymore, so no need to enable it on projects, that not use `logger_entity`.